content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-client-extensions-model.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Add custom methods to your models (Prisma Client extensions)'
+metaTitle: 'Prisma Client extensions: model (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+This page is in draft.
+
+</Admonition>
+
+From version **TBD**, you can use `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add custom methods to your models.
```

Comment:
Can one really just add methods, or could one also add other constructs like properties?

Reply:
Yep, this is something we need to clarify. I'll ask about this in the dedicated channel

Reply:
It looks like properties might be possible with the `client` extension type, but we haven't tested this yet, so we can add this in a future interaction of the docs